mmpose/models/keypoint_heads/fc_head.py

```diff
@@ -115,6 +115,25 @@ def inference_model(self, x, flip_pairs=None):
             output_regression = output.detach().cpu().numpy()
         return output_regression
 
+    def decode(self, img_metas, output, **kwargs):
```

Comment:
decode_heatmap?

Reply:
We will support multi-task network with multiple heads for depth estimation, classification, and keypoint estimation soon. To make a unified interface for different heads, we will add a `decode` function for all types of heads. 

For keypoint head, `decode` means `decode_heatmap`; while for depth head, `decode` means `decode_depthmap` etc.

Reply:
How to determine which `decode_xx` to call?

Reply:
I'm trying to understand the design philosophy here. So if you could provide more details...

Reply:
Okay, as discussed offline, we rename all decode_xx into decode. There will be no multiple decodes in one class.